### DISCUSSION TITLE
RuntimeError: do not find download file frozen_model.pb

### DISCUSSION THREAD

**Discussion by komorebi566** [deepmodeling/dpgen#1011] [Q&A] [answered]
Hello，I had some problems during freezing the frozen_model.pb
An error is HYDU_create_process (utils/launch/launch.c:74): execvp error on file export (No such file or directory),
I guess something went wrong with the order: 
mpirun -machinefile $PBS_NODEFILE -n 1 export CUDA_VISIBLE_DEVICES=0 && dp freeze  1>> train.log 2>> train.log
My  dpgen version is 0.10.6 
 numpy     1.22.3 
dpdata      0.2.7
pymatgen            unknown version or path
monty  2022.4.26
ase     3.22.1
paramiko      2.8.1
 custodian  2022.5.26
Thanks for any help~

**Comment by njzjz** [comment, marked as answer]
I don't know how you generated such command, but the correct way to run multiple commands with mpi is

```sh
mpirun -machinefile $PBS_NODEFILE -n 1 sh -c "export CUDA_VISIBLE_DEVICES=0 && dp freeze" 1>> train.log 2>> train.log
```

**Reply by komorebi566** [reply]
I found this command in the .sub file generated by dpgen.
But I'm sorry that I cannot show some details for you, because I changed the command "with_mpi": false in machine.json, and it was successfully frozen.
